test/integration/full-content/fixtures/core__columns.parsed.json

```diff
@@ -13,16 +13,29 @@
                         "blockName": "core/paragraph",
                         "attrs": {},
                         "innerBlocks": [],
-                        "innerHTML": "\n\t\t<p>Column One, Paragraph One</p>\n\t\t"
+                        "innerHTML": "\n\t\t<p>Column One, Paragraph One</p>\n\t\t",
+                        "innerContent": [
+                            "\n\t\t<p>Column One, Paragraph One</p>\n\t\t"
+                        ]
                     },
                     {
                         "blockName": "core/paragraph",
                         "attrs": {},
                         "innerBlocks": [],
-                        "innerHTML": "\n\t\t<p>Column One, Paragraph Two</p>\n\t\t"
+                        "innerHTML": "\n\t\t<p>Column One, Paragraph Two</p>\n\t\t",
+                        "innerContent": [
+                            "\n\t\t<p>Column One, Paragraph Two</p>\n\t\t"
+                        ]
                     }
                 ],
-                "innerHTML": "\n\t<div class=\"wp-block-column\">\n\t\t\n\t\t\n\t</div>\n\t"
+                "innerHTML": "\n\t<div class=\"wp-block-column\">\n\t\t\n\t\t\n\t</div>\n\t",
+                "innerContent": [
+                    "\n\t<div class=\"wp-block-column\">\n\t\t",
+                    null,
+                    "\n\t\t",
```

Comment:
Trying to think if the serializer behavior of newline-delimited blocks could have an impact in the ability to re-serialize, but I'm assuming the editor will disregard this as far as parsing/rehydrating the top-level columns block (which does and will continue to operate on `innerHTML`?).

Reply:
good point. we may have to take some additional care to make sure that the serializer matches behavior with the parser, but since it's been working so far I'm guessing it will continue to work for a while.